CRM/Event/Form/Participant.php

```diff
@@ -2235,6 +2255,7 @@ protected function sendReceipts($params, $total_amount, array $customFields, arr
         'modelProps' => [
           'participantID' => $this->_id,
           'eventID' => $params['event_id'],
+          'contributionId' => $this->getContributionID(),
```

Comment:
should it be ID or Id?

Reply:
Unfortunately this consistency goes deeper at the moment - I've been thinking about whether I can sort it out - but not in this PR